sdkSeedProviders.json

```diff
@@ -1,12 +1,24 @@
 {
-  "testnet": [
-    {
-      "rpcAddress": "providers-us-1.lavanet-zarak.xyz:20006",
-      "publicAddress": "lava@1vfpuqq06426z3x4qsn38w6hdqrywqxlc6wmnxp"
-    },
-    {
-      "rpcAddress": "providers-us-1.lavanet-zarak.xyz:19927",
-      "publicAddress": "lava@12u7dam8tyedr82ntwe6zz5e34n6vhr3kjlanaf"
-    }
-  ]
+  "testnet": {
+    "us": [
```

Comment:
lets call it North-America

Reply:
Fixed https://github.com/lavanet/lava-providers/pull/6/commits/6981891cb09354cc6b8e2383b61a04178bbe3ed4